stories/core/UserIconGroup.stories.tsx

```diff
@@ -0,0 +1,322 @@
+    'Morgan Vera',
+    'Jean Mullins',
+    'Ashley Miles',
+  ];
+
+  return (
+    <UserIconGroup animated {...args}>
+      {userNames.map((name) => (
+        <UserIcon
+          key={name}
+          abbreviation={name
+            .split(' ')
+            .map((token) => token[0])
+            .join('')}
+          backgroundColor={getUserColor(name)}
+          title={name}
+        />
+      ))}
+    </UserIconGroup>
+  );
+};
+
+Animated.args = {
+  animated: true,
+  iconSize: 'medium',
+};
+
+export const ManyIcons: Story<UserIconGroupProps> = (args) => {
+  const userNames = [
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ace Cash',
+    'Tanner Fraser',
+    'Ashley Miles',
+    'Jean Mullins',
+    'Nico Triplett',
+    'Drew Abel',
+    'Kendall Simons',
+    'Kennedy Gray',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Jessie Dodd',
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ace Cash',
+    'Tanner Fraser',
+    'Ashley Miles',
+    'Jean Mullins',
+    'Nico Triplett',
+    'Drew Abel',
+    'Kendall Simons',
+    'Kennedy Gray',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Jessie Dodd',
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ace Cash',
+    'Tanner Fraser',
+    'Ashley Miles',
+    'Jean Mullins',
+    'Nico Triplett',
+    'Drew Abel',
+    'Kendall Simons',
+    'Kennedy Gray',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Jessie Dodd',
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ace Cash',
+    'Tanner Fraser',
+    'Ashley Miles',
+    'Jean Mullins',
+    'Nico Triplett',
+    'Drew Abel',
+    'Kendall Simons',
+    'Kennedy Gray',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Jessie Dodd',
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ace Cash',
+    'Tanner Fraser',
+    'Ashley Miles',
+    'Jean Mullins',
+    'Nico Triplett',
+    'Drew Abel',
+    'Kendall Simons',
+    'Kennedy Gray',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Jessie Dodd',
+    'Kennedy Gray',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Jessie Dodd',
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ace Cash',
+    'Tanner Fraser',
+    'Ashley Miles',
+    'Jean Mullins',
+    'Nico Triplett',
+    'Drew Abel',
+    'Kendall Simons',
+    'Kennedy Gray',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Jessie Dodd',
+    'Terry Rivers',
+    'Robin Mercer',
+    'Morgan Vera',
+    'Ace Cash',
+    'Tanner Fraser',
+    'Ashley Miles',
+    'Jean Mullins',
+    'Nico Triplett',
+    'Drew Abel',
+    'Kendall Simons',
+    'Kennedy Gray',
+    'Charlie Mayfield',
+    'Peyton Pennington',
+    'Justice Harrington',
+    'Jessie Dodd',
+  ];
```

Comment:
If you're just repeating the names, you don't really need an array that's so long.

You can just define the unique ones, and iterate over them using `%`.